src/store/notifications.js

```diff
@@ -17,7 +17,9 @@ const REMOVE_NOTIFICATION = 'REMOVE_NOTIFICATION';
  * @returns {Object} notification
  */
 export const addNotification = (notification) => (dispatch) => {
-  notification.id = new Date().getTime();
+  if (notification.id === undefined) {
```

Comment:
use `if (!notification.id) {` here, it will prevent empty string, `null` case, etc...
As you can see, L21 is no more covered because we set `notification.id` to `null` in the test case : https://coveralls.io/builds/8020458/source?filename=src%2Fstore%2Fnotifications.js#L21